<?xml version="1.0"?>
<launch>
 
  <!-- Launch parameters -->
  <arg name="output" default="screen" /> <!-- screen/log -->
  <arg name="required" default="true" />
  <arg name="respawn" default="false" />

  <!-- Node parameters --> 
  <node pkg="mkz_control" type="mkz_control" name="mkz_control" respawn="$(arg respawn)" required="$(arg required)" output="$(arg output)">
    
    <param name="max_ey"                 value="1.5" /> <!-- m -->
    <param name="max_ephi"               value="45" />  <!-- degree -->
    <param name="gain_folder"            value="/home/mtl/Haojie/mkz_controller_drivers/src/preview_control/data/gain/withoutdelay/" />

    <param name="speed_ctrl_kp"          value="1.3" />
    <param name="speed_ctrl_ki"          value="0.5" />
 
  </node>
</launch>
